modules/nf-neuro/image/burnvoxels/tests/main.nf.test.snap

```diff
@@ -0,0 +1,61 @@
+{
+    "burnvoxels - one mask": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        "test__all.nii.gz:md5,7e8fe36daf0dfba28a5701c55e98a0ab"
+                    ]
+                ],
+                "1": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        [
+                            "test__IFGWM.nii.gz_25.nii.gz:md5,d9654356d7f9c2b0eda5f69e8e71b9c4",
+                            "test__IFGWM_labels_map.nii.gz_325.nii.gz:md5,1169f8f2dba2ef2addaea6afd0ee82c2"
```

Comment:
here you have an issue with the extension

Reply:
Good catch. Will patch this.